tests/commands/base/test_verify_tools.py

```diff
@@ -1,3 +1,10 @@
+from briefcase.integrations.download import Download
+from briefcase.integrations.subprocess import Subprocess
+
+
 def test_base_tools_exist(base_command):
-    """No verification required for base command."""
+    """Ensure default tools are always available."""
+    assert isinstance(base_command.tools.subprocess, Subprocess)
+    assert isinstance(base_command.tools.download, Download)
```

Comment:
This test has lost the check of `app` binding in the port from test_ToolCache.py - it looks like that might also be a side effect of moving the registration to base. Was this intentional? Is it a source of concern that the app subclasses don't inherit these tools any more?